### ISSUE TITLE
accessibility - buttons in the challenge page are not accessible via keyboard

### ISSUE THREAD

**Issue by sarbbottam** [FreeCodeCamp/FreeCodeCamp#8590]
#### Challenge Name
Any

#### Issue Description
Try pressing the `tab` key to reach the `buttons`, they never receives focus.
Currently the buttons are implemented as `div`, they should be `button`.

#### Browser Information
* Browser Name, Version: Any
* Operating System: Any
* Desktop & Mobile & Tablet

#### Screenshot
<img width="1280" alt="screen_shot_2016-05-12_at_3_44_29_pm" src="https://cloud.githubusercontent.com/assets/949380/15232858/d7f9fd96-1858-11e6-8c6b-bc776a45c6a7.png">

**Comment by raisedadead** [comment]
@sarbbottam Thanks for the issue and the PR.

PS : I have a recommendation, in case you are going to make a PR for any issue that you find yourself, then its not necessary to raise a issue separately. You can directly make a pull request :wink: and discuss any concerns on that thread! 

Again thanks a lot.

**Comment by sarbbottam** [comment]
Point noted 😄 , Thanks!!
